ui/app/styles/lab.scss

```diff
@@ -0,0 +1,47 @@
+.lab\/header {
+  padding-bottom: scale.$base;
+  margin-bottom: scale.$base;
+  border-bottom: 1px solid rgb(var(--border));
+}
+
+.lab\/runner-table {
```

Comment:
😅 good thing this is a lab feature because I wouldn't want to generalize `/`-based classnames

Reply:
haha, yeah, this is the lab, where we get to try out weird stuff!